### DISCUSSION TITLE
Smartagent/jmx receiver break kubernetes integration.

### DISCUSSION THREAD

**Discussion by Mironiken** [signalfx/splunk-otel-collector#2242] [Q&A]
Hello,
I am using Splunk Distribution of OpenTelemetry Collector in kubernetes. Current solution is working just fine. But after I added section for the smartagent/jmx receiver with groovy script inside, healthcheck starts to show "Server not available" status.  Groovy script works, logs includes several
2022-11-07T11:12:14.730Z error subproc/core.go:114 Get result, and sent:0 {"kind": "receiver", "name": "smartagent/jmx", "pipeline": "metrics", "monitorID": "smartagentjmx", "monitorType": "jmx", "runnerPID": 42}
(I just added stderr to script)

There are not any other warn/errors in logs. Kubernetes just kills pod cause healthcheck.  My jmx config:
```
smartagent/jmx:
  type: jmx
  host: hazelcast_address
  port: 9999
  intervalSeconds: 2
  groovyScript: |
    def printErr = System.err.&println
    ss = util.queryJMX("com.hazelcast:name=MAP_NAME,instance=*,type=IMap").first()
    dims = [env_name: "NAME"]
    output.sendDatapoint(util.makeGauge("hazelcast.map.size", ss.size, dims))
    printErr("Get result, and sent:" + ss.size)
```
And - smartagent/jmx added into service/pipelines/metrics/receivers section.
Do you have any ideas where and how to dig?

**Comment by atoulme** [comment]
This is odd. I recommend you read through our troubleshooting instructions and try to set log level to debug:
https://github.com/signalfx/splunk-otel-collector/blob/main/docs/troubleshooting.md
